spec/_attachments/ic.did

```diff
@@ -184,6 +192,14 @@ service ic : {
   bitcoin_send_transaction: (send_transaction_request) -> ();
   bitcoin_get_current_fee_percentiles: (get_current_fee_percentiles_request) -> (vec millisatoshi_per_byte);
 
+  // metrics interface
+  metrics : (record {
+    subnet_id : principal;
+  }) -> (record {
+    timestamp_nanos : nat64;
+    metrics : vec metrics;
```

Comment:
```suggestion
    subnet_metrics : vec node_metrics;
```

Reply:
Shouldn't this be called node_metrics then?